src/ui/css/styles.scss

```diff
@@ -1,2 +1 @@
-@import '~core/css/inc/mixins';
-@import './vars';
+@import '~amo/css/styles';
```

Comment:
I don't think we want to do this because `amo` and `ui` should be kept separated.

Reply:
So should the styles in `ui` be changed in `ui/css/styles` and those in `amo` in `amo/css/styles`?

Reply:
Yes